test/nadia_test.exs

```diff
@@ -5,10 +5,7 @@ defmodule NadiaTest do
   alias Nadia.Model.User
 
   setup_all do
-    unless Application.get_env(:nadia, :token) do
-      Application.put_env(:nadia, :token, "TEST_TOKEN")
-    end
-    ExVCR.Config.filter_sensitive_data("bot[^/]+/", "bot<TOKEN>/")
+    Application.put_env(:nadia, :token, "<TOKEN>")
```

Comment:
Since we can also set config at `config/test.exs`, so I set the token only when it's missing.
Maybe it's better to remove the setting here, and use `config/test.exs` only.

Anyway, changes here does not affect the support of multiple token which is the main purpose of this p-r.
So I prefer to fix it in a new p-r.